introduction.md

```diff
@@ -6,33 +6,44 @@ layout: documentation
 
 # About openHAB
 
-openHAB is **software** for integrating different home automation systems and technologies into one single solution that allows over-arching automation rules, and that offers uniform user interfaces.
+openHAB **software** integrates different home automation systems, devices and technologies into a single solution.
+It provides uniform user interfaces, and a common approach to automation rules across the entire system, regardless of the number of manufacturers and sub-systems involved.
 
-But openHAB is also a **community** of users, contributors and maintainers.
+## Getting Started
 
-## openHAB Structure
-
-openHAB 2 is an open-source solution based on the [Eclipse SmartHome](https://eclipse.org/smarthome/) framework. It is written in Java and uses [Apache Karaf](http://karaf.apache.org/) together with [Eclipse Equinox](https://www.eclipse.org/equinox/) as an OSGi runtime and bundles this with [Jetty](https://www.eclipse.org/jetty/) as an HTTP server.
+Now that you know what openHAB is, you may be wondering how to get started.
+We highly recommend that you read the next article titled [Concepts]({{base}}/concepts/index.html).
+This article introduces a number of important ideas that will help you as you install and begin to configure openHAB for the first time.
+openHAB runs on many popular platforms including Linux, Windows and Mac OSx.
+You can find specific installation instructions for these and other platforms in the [Installation Overview]({{base}}/installation/intex.html) article.
+Many people find that the simplest way to experiment with openHAB and learn what it can do without making a significant investment is to get a [Raspberry Pi](https://raspberrypi.org) and install [openHABian]({{base}}/installation/openhabian.html), a "hastle-free openHAB setup".
+While openHABian offers a streamlined and simplified way to get up and running quickly, it is a complete openHAB home automation system capable of automating your entire home.
```

Comment:
I like it!

Not sure if we need to mention openHABian here. It's discussed in the Installation Overview.

Reply:
@ThomDietrich, I will do whatever you think is best.  But I like mentioning it here because I expect people who are reading this for the first time are "new customers".  I want to be sure they know right at the beginning that this project has specifically created an "on-ramp" for them that makes it easy to get up and running.
One of the biggest challenges for any home automation setup is to figure out how to make it easy for people to get up and running quickly.  Developers have done a great job of this with openHABian, and I wanted to promote it up front.

Let me know what you prefer.

Reply:
I agree with @bgilmer77.
It is a good hint for beginners and helps up a first environment in a comfortable way.

Reply:
Unless I hear otherwise from @ThomDietrich, I am going to leave openHABian in my next commit.

Reply:
Okay then 😃

Reply:
This comment has been addressed.